parsec/core/fs/remote_loader.py

```diff
@@ -247,7 +249,7 @@ async def upload_block(self, access: BlockAccess, data: bytes):
     async def load_manifest(
         self,
         entry_id: EntryID,
-        version: int = None,
+        version: Optional[int] = None,
         timestamp: Pendulum = None,
```

Comment:
optional on timestamp and expected_backend_timestamp too

Reply:
fixed